core-java-modules/core-java-security-4/src/main/java/com/baeldung/enablessldebug/SSLDebugLogger.java

```diff
@@ -0,0 +1,7 @@
+package com.baeldung.enablessldebug;
+
+public class SSLDebugLogger {
+    public static void enableSSLDebugUsingSystemProperties() {
+        System.setProperty("javax.net.debug", "ssl:debug");
```

Comment:
Would your tests still work if we moved this snippet of code into a private method within the Unit test class? I'm thinking that in that case we could remove this source code class altogether.

At the same time we want to replicate the setup someone would have in their project.

What are your thoughts?

Reply:
"At the same time we want to replicate the setup someone would have in their project."
Didn't get this, could you please clarify?